dashboard/common/db_util.py

```diff
@@ -21,16 +23,16 @@ def canvas_id_to_incremented_id(canvas_id):
     return int(canvas_id) + settings.CANVAS_DATA_ID_INCREMENT
 
 
-def incremented_id_to_canvas_id(incremented_id):
+def incremented_id_to_canvas_id(incremented_id: Union[int, str]) -> Optional[int]:
     try:
         int(incremented_id)
     except ValueError:
         return None
 
-    return str(int(incremented_id) - settings.CANVAS_DATA_ID_INCREMENT)
+    return int(incremented_id) - settings.CANVAS_DATA_ID_INCREMENT
```

Comment:
Did you test the cron to make sure this doesn't break anything? It looks like `incremented_id_to_canvas_id` is only used in `update_with_bq_access`. The `bigquery.ArrayQueryParameter` on line 263 seems to be expecting a list of strings, though it seems like the line above it may be wrong also (since it says `STRING` but `data_warehouse_course_ids` should be a list of integers.

Reply:
I haven't used those methods before, so it's entirely possible none of this is an issue.

Reply:
No, I only tested the UI, I'll test the cron. With the return type being int now and CANVAS_DATA_ID_INCREMENT also being an int I figured this cast wasn't needed  but I'll try it.

Reply:
I just tested the cron and it's running fine for me.